app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<section>
+  <div class='one-user'>
+    <div class='user-photo'>
+      <img src='<%= @user.photo %>' alt='<%= @user.name %> thumbnail' />
+    </div>
+    <div class='user-name'>
+      <h2><%= @user.name %></h2>
+      <p>Number of Posts: <%= @user.posts_counter || 0 %></p>
+    </div>
+  </div>
+  <div class='user-bio'>
+    <h4>Bio</h4>
+    <h5><%= @user.bio %></h5>
+  </div>
+  <div class='posts'>
+    <% @user.posts.each do |post| %>
```

Comment:
Here, you are using `recent_post` method to render the user's 3 most recent posts. Please use 
`<% @user.recent_post.each do |post| %>` code instead of   `<% @user.post.each do |post| >`

Reply:
Thanks @OLIPLICHE . I was able to implement this and everything works fine.